dioptas/model/MaskModel.py

```diff
@@ -231,24 +232,44 @@ def set_mask(self, mask_data):
         self.update_deque()
         self._mask_data = mask_data
 
-    def save_mask(self, filename):
+    def save_mask(self, filename, flipud: bool=False):
```

Comment:
Adding a `flipud` argument so the mask model does not need access to the image model.